DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/wallet/applepay/refund-apple-pay.md

>_Outgoing payments will not be processed if your MultiSafepay balance is insufficient._

> In the event of a failed Apple Pay refund, contact our Support Team at <support@multisafepay.com> to assist you.

Reviewer: ```suggestion
If an Apple Pay refund fails, email the Support Team at <support@multisafepay.com>
```
